### DISCUSSION TITLE
Unable to figure out Bluetooth password

### DISCUSSION THREAD

**Discussion by ElvezPelvez** [lkempf/casambi-bt-hass#67] [Q&A]
Installation went fine (I guess). However, trying to configure the integration I am being asked for the Bluetooth MAC address and password. I checked everything in the Casambi app (iOS) but I'm not able to find anything that would resemble a password.

Any ideas how I could figure this out?
Thanks in advance!

**Comment by lkempf** [comment]
Normally your network should get discovered automatically. If that isn't working see https://github.com/lkempf/casambi-bt-hass/issues/64#issuecomment-1948218129 for instructions 

The password is something you added when setting up the network. I think there is a passwordless mode but that isn't supported. See https://github.com/lkempf/casambi-bt?tab=readme-ov-file#casambi-network-setup for that

**Reply by ElvezPelvez** [reply]
Thanks a lot for your quick help…

I just figured out I seem to be using the wrong type of network (classic instead of evolution). Unfortunately, upgrading all my devices is not an option so I guess I‘ll have to sit and wait until someone figures out how to connect to the classic network…
